runtime/compiler/z/codegen/J9MemoryReference.cpp

```diff
@@ -393,14 +397,15 @@ J9::Z::MemoryReference::create(TR::CodeGenerator* cg, TR::Node* node)
 
       typedef J9::Z::UnresolvedDataReadOnlySnippet::CCUnresolvedData CCUnresolvedData;
 
-      CCUnresolvedData* ccUnresolvedDataAddress =
-         reinterpret_cast<J9::Z::UnresolvedDataReadOnlySnippet::CCUnresolvedData*>(cg->allocateCodeMemory(sizeof(CCUnresolvedData), false));
-
-      if (ccUnresolvedDataAddress == NULL)
+      OMR::CCData *codeCacheData = cg->getCodeCache()->manager()->getCodeCacheData();
+      OMR::CCData::index_t index;
+      if (!(codeCacheData->put(NULL, sizeof(CCUnresolvedData), 8, NULL, index)))
          {
-         comp->failCompilation<TR::CompilationException>("Could not allocate unresolved data metadata");
+         comp->failCompilation<TR::CompilationException>("Could not allocate uVdnresolved data metadata");
```

Comment:
New typo: uVdnresolved

Reply:
fixed in https://github.com/eclipse/openj9/compare/e6572d23ab0cf110a98cc239bea656e8aa8729ec..6d1fee3c7c3d3066a711500dc5c93851f9df074c